deployments/examples/ocis_full/.env

```diff
@@ -203,9 +203,27 @@ WOPISERVER_ONLYOFFICE_DOMAIN=
 # Domain for Inbucket. Defaults to "mail.owncloud.test".
 INBUCKET_DOMAIN=
 
+### LDAP server settings ###
```

Comment:
If it is openldap (openldap.yml), name it openldap

Reply:
Can do, I just cut and paste from https://github.com/owncloud/ocis/blob/0fa0c5e0b1f26ac03e6c097e48bae4517f31e156/deployments/examples/ocis_ldap/.env#L31